Allow transformed URL to sign (#181)
uploadcare/uploadcare-php: CHANGELOG.md

Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to
[Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [3.1.1]
### Secure URL generation update

> - Added possibility to generate Security CDN url for image with transformations.

Reviewer: ```suggestion
- Now you can generate Secure CDN URLs for images with transformations.
```
